hack/docs/generate_samples.go

```diff
@@ -58,6 +59,12 @@ func updateTutorial(generator tutorial_generator) {
 	generator.CodeGen()
 }
 
+func UpdateMultiversionTutorial() {
+	samplePath := "docs/book/src/multiversion-tutorial/testdata/project/"
+	sp := multiversion.NewSample(KubebuilderBinName, samplePath)
```

Comment:
Hi @sarthaksarthak9,

I was thinking about this approach:

- First, we need to call the CronJob tutorial generation
- Then we need to add on top what is required to do the conversion of one version to another

See that docs explain how to set up the CronJob Tutorial, and after that, it  uses the tutorial to scaffold and add the code needed to explain how to migrate from one version to another.

Therefore, to avoid duplications and we are able to keep both easily maintained we would need to the same.
WDYT? Is that something that make sense?

Reply:
PLease, see: https://github.com/kubernetes-sigs/kubebuilder/pull/4138

Reply:
@camilamacedo86  Thank you your pointers. Yes, I agree with this.